### ISSUE TITLE
httpHeader is not sent to the server (authorization failed)

### ISSUE THREAD

**Issue by javatutorials2016** [hyochan/react-native-audio-recorder-player#400]
> Please fill the template to help you out. Also, please try the `Example` project compare before submiting the issue when you have certain issue with your project setup.

### Version of react-native-audio-recorder-player
"react-native-audio-recorder-player": "^3.2.0"

### Version of React Native
 "react-native": "0.65.1"
### Platforms you faced the error (IOS or Android or both?)
IOS (emulator)
### Expected behavior
The audio url to be streamed.
### Actual behavior
Not playing from the URL. At the server can see that authorization failed because the authorization header is not received.

### Steps to reproduce the behabior
 const msg = await this.audioRecorderPlayer.startPlayer(this.props.audioUrl, { 'Authorization': `Bearer ${this.props.authorization}` });

**Comment by hyochan** [comment]
I've not had time to implement this feature yet. I need to search for `iOS` spec on where to pass the header to when playing from a remote URL.

**Comment by hyochan** [comment]
I'll try to resolve this issue soon @javatutorials2016. Looks like we need to pass header with `AVURLAssetHTTPHeaderFieldsKey`.

```
 let httpHeader: [String: String] = [
    "key": "value"
 ]
 let asset = AVURLAsset(url: URL, options: ["AVURLAssetHTTPHeaderFieldsKey": httpHeader])
 let playerItem = AVPlayerItem(asset: asset)
 player = AVPlayer(playerItem: item)
```

You may also try for `PR` if you could.

**Comment by javatutorials2016** [comment]
![Screen Shot 2021-11-15 at 8 23 36 AM](https://user-images.githubusercontent.com/19355588/141792140-d1988bb3-31b7-4d13-b56b-d1ec538b8d09.png)

will do as soon as I resolve the attached issue, however, If you have it already done, please merge.

**Comment by hyochan** [comment]
Closes #405
